### PR TITLE
Supports building windows server 2019

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
Need to update the codebuild job to run win19 before testing...

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

File: main.tf [hunk 1]
```diff
@@ -17,7 +17,7 @@ resource "aws_instance" "win_builder" {
   vpc_security_group_ids = [join("", aws_security_group.winrm_sg.*.id)]
 
   tags = {
-    Name = "${local.resource_name}-builder"
+    Name = "${local.resource_name}-builder-${local.ami_underlying[local.win_builder_ami_key]}"
```
Comment (YakDriver) [inline main.tf]: Why remove this?

Comment (lorengordon) [reply main.tf]: It's not removed, just updating the name tag with more info to make it easier to see what is building in the console.

Comment (YakDriver) [reply main.tf]: Oh, yes. Sorry confused the removing with the adding!

**Pull request merged**